.github/workflows/prs.yml

```
@@ -422,7 +424,7 @@ jobs:
     name: Compute K8s test plan
     runs-on: ubuntu-latest
     needs: changes
-    if: ${{ needs.changes.outputs.source == 'true' }}
+    if: ${{ needs.changes.outputs.source == 'true' }} || ${{ needs.changes.outputs.helm == 'true' }} || ${{ needs.changes.outputs.docker == 'true' }}
```

```suggestion
    if: needs.changes.outputs.source == 'true' || needs.changes.outputs.helm == 'true' || needs.changes.outputs.docker == 'true'
```

From the https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions:

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. For more information about if conditionals, see "Workflow syntax for GitHub Actions."

I think the `${{ }}` and `||` like it was are a syntax error.

---

If they were a syntax error the job would have failed to run. Removing the expression syntax is optional.